StripeFinancialConnections/StripeFinancialConnections/Source/Common/FlowRouter.swift

```diff
@@ -33,13 +43,12 @@ class FlowRouter {
     // MARK: - Public
 
     var flow: Flow {
-        guard synchronizePayload.manifest.isProductInstantDebits == false else {
-            // We currently only support Instant Debits via a web flow.
-            return .webInstantDebits
+        if synchronizePayload.manifest.isProductInstantDebits {
```

Comment:
I also felt like if-statement is better here than guard because its not really a "precondition" - its a whole different conditional/flow

Reply:
Exactly!